UsersGitHub/UsersGitHub/ViewModel/ReposViewModel.cs

```diff
@@ -29,14 +30,15 @@ public ObservableCollection<Repository> Repositories
             }
         }
 
-        public ReposViewModel()
+        public ReposViewModel(User user)
         {
+            this.user = user;
             GetUrerRepositories();
         }
 
         private async void GetUrerRepositories()
         {
-            Repositories = (await BlobCache.UserAccount.GetAllObjects<User>()).Last().Repositories;
+            Repositories = (await BlobCache.UserAccount.GetObject<User>(user.UserName)).Repositories;
```

Comment:
so you are downloading user repos only once when you add user
what if user will create another repository in meantime? it will never be shown in our application?